packages/shared/lib/services/file/remote.service.ts

```diff
@@ -11,29 +11,52 @@ import { NangoError } from '../../utils/error.js';
 import errorManager from '../../utils/error.manager.js';
 
 import type { ServiceResponse } from '../../models/Generic.js';
-import type { GetObjectCommandOutput } from '@aws-sdk/client-s3';
+import type { GetObjectCommandOutput, S3ClientConfig } from '@aws-sdk/client-s3';
 import type { DBSyncConfig } from '@nangohq/types';
 import type { Response } from 'express';
 
+function getCredentials() {
+    const accessKeyId = process.env['AWS_INTEGRATIONS_ACCESS_KEY_ID'];
+    const secretAccessKey = process.env['AWS_INTEGRATIONS_SECRET_ACCESS_KEY'];
+    if (!accessKeyId || !secretAccessKey) {
+        return undefined;
+    }
+    return {
+        accessKeyId,
+        secretAccessKey
+    };
+}
```

Comment:
The names of the AWS credentials environment variables have to be changed because the permissions are provided to the runtime environment at an infrastructure level - keeping the environment variables named as the default (i.e. `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY_ID`) overrides any profile assumed by the runtime. The credential provider for AWS evaluates environment variables to have higher priority.